### PR TITLE
DAOS-5156 test: Add monitoring and health checks to soak

### PR DESCRIPTION
**Pull request by mjean308**
Skip-unit-tests: true
Skip-func-hw-small: true
Skip-func-hw-test-medium: true
Test-tag-hw-large: soak_smoke

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

**Review by shimizukko — APPROVED** [review]
LGTM

**Review by shimizukko — APPROVED** [review]
LGTM

File: src/tests/ftest/slurm_setup.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 PACKAGE_LIST = ["slurm", "slurm-example-configs",
                 "slurm-slurmctld", "slurm-slurmd"]
 
+PACKAGE_VERSION = "18.08.8-1.el7.x86_64"
```
Comment (saurabhtandan) [inline src/tests/ftest/slurm_setup.py:23]: Minor: Is it possible to automate it? We will have to change it whenever it is updated.

**Review by saurabhtandan — APPROVED** [review]
LGTM

File: src/tests/ftest/slurm_setup.py (hunk 1 above)
Comment (mjean308) [reply src/tests/ftest/slurm_setup.py:23]: It has recently been updated to version 20 and it broke the install.  However, v20 slurm now has a dependency on hdf5 and they are using an older version (v 1.8 ) and so we would need to rebuild slurm 20 from source in order to use in our env.  slurm 18 is still available.

**Pull request merged**